content/en/continuous-deployment/release-notes/rn-armory-spinnaker/armoryspinnaker_v2-28-7.md

```diff
@@ -0,0 +1,253 @@
+---
+title: v2.28.7 Armory Release (OSS Spinnaker™ v1.28.0)
```

Comment:
```suggestion
title: v2.28.7 Armory Release
```

Reply:
Suggesting this because we no longer base our release on an OSS release, we base them on individual service updates